var/spack/repos/builtin/packages/r-dtplyr/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RDtplyr(RPackage):
+    """Data Table Back-End for 'dplyr'.
+    Provides a data.table backend for 'dplyr'. The goal of 'dtplyr' is to
```

Comment:
Add a blank line after the first.
```suggestion
    """Data Table Back-End for 'dplyr'.
    
    Provides a data.table backend for 'dplyr'. The goal of 'dtplyr' is to
```